scripts/ci/pre_commit/pre_commit_check_provider_yaml_files.py

```diff
@@ -151,7 +151,10 @@ def parse_module_data(provider_data, resource_type, yaml_file_path):
     package_dir = ROOT_DIR + "/" + os.path.dirname(yaml_file_path)
     provider_package = os.path.dirname(yaml_file_path).replace(os.sep, ".")
     py_files = chain(
-        glob(f"{package_dir}/**/{resource_type}/*.py"), glob(f"{package_dir}/{resource_type}/*.py")
+        glob(f"{package_dir}/**/{resource_type}/*.py"),
+        glob(f"{package_dir}/{resource_type}/*.py"),
+        glob(f"{package_dir}/**/{resource_type}/**/*.py"),
+        glob(f"{package_dir}/{resource_type}/**/*.py"),
```

Comment:
I have done this changes, because for Vertex AI we use package for code rather than one module and for packages our static check doesn't work.